### DISCUSSION TITLE
Clip Art not rendering correctly

### DISCUSSION THREAD

**Discussion by gordon-brown** [clientIO/joint#2464] [Q&A]
### Introduction

I'm having an issue rendering one of the clip art shapes I converted. It seems to me to be related to rendering elliptical curves in the path data. I can render the shape on the https://www.svgviewer.dev/ site. Not sure if I am doing something wrong or if this is an  issue with your software.
[BPMN_StoredData-202401030954.joint.txt](https://github.com/clientIO/joint/files/13820532/BPMN_StoredData-202401030954.joint.txt)
![Screenshot 2024-01-03 100103](https://github.com/clientIO/joint/assets/32515524/088c92a0-aefe-415b-9b5b-bf24454ee7de)
![Screenshot 2024-01-03 100348](https://github.com/clientIO/joint/assets/32515524/b90d978b-b8c9-48e6-acbc-8ec4f9060e5a)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
There seems to be an issue with an `arc` command that has zero length and our code is not able to convert it to the `curve` command. I am changing the question to a bug.

It works if you define your path like this: 'M 5.555555555555555 0L 50 0A 5.555555555555555,25 0 0,0 50,50 L 50,50L 5.555555555555555 50A 5.555555555555555,25 0 0,1 5.555555555555555,0Z'

**Reply by gordon-brown** [reply]
Thanks. I can update the path data in the interm. We only have a couple of shapes like this.

**Reply by zbynekstara** [reply]
Hi @gordon-brown, the issue with zero-length arcto curves was resolved as part of JointJS v4.0.2 (JointJS+ v4.0.1)
